tests/test_schema.yaml

```diff
@@ -1655,7 +1655,7 @@ schemas:
   prefix: /v2.0
   schema:
     indexes:
-      test_unique_id_and_tenant_id_and_domain_id:
+      custom_action_test_unique_id_and_tenant_id_and_domain_id:
```

Comment:
why is this changed?

Reply:
to fix failing tests on SQLite. Current master is broken.